pipelines/ocw/scripts/pipeline-ocw-www.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This script requires prior authentication via target "odl-concourse"
+
+# The following variables need to be set, and will be different between RC and production:
+# GIT_CONFIG_ORG=<the github organization>
+# GITHUB_DOMAIN=<the github domain>
+# OCW_WWW_REPO=<the github repo for the ocw-www site>
+# OCW_CONTENT_BUCKET_PREVIEW=<the S3 preview bucket>
+# OCW_CONTENT_BUCKET_RELEASE=<the S3 release bucket>
+# OCW_STUDIO_BASE_URL=<the ocw-studio base url>
+
+# The following need to be set via vault, per environment and pipeline group (preview, release):
+# fastly-service-id
+# fastly-api-token
+# git-private-key
+
```

Comment:
@blarghmatey I incorporated groups into the scripts: there's now a `ocw-theme-assets` group with `release` and `release-candidate` branches, plus a `ocw-www-pipeline-via-vault` group with `preview` and `release` branches.

I removed the assignment of fastly & github key values from the ocw-www pipeline script so that it will try to pick them up from vault.  But I assume they are not there yet because those new pipelines are failing.

There are other values that will be different between RC and production, as documented above.  The values for  some of them  are yet to be determined for production.  So I think environment values would still be best for those?

Also, I'm still not sure how best to handle authentication, I'm unclear where these scripts are intended to be run from. For now I'm assuming authentication has already been taken care of, with fly target `odl-concourse`.